src/config/configuration.ts

```diff
@@ -114,7 +114,7 @@ const configuration = () => {
     httpMaxRedirects: process.env.HTTP_MAX_REDIRECTS ?? 5,
     httpTimeOut: process.env.HTTP_TIMEOUT ?? 5000,
     jwt: {
-      secret: process.env.JWT_SECRET,
+      secret: process.env.JWT_SECRET ?? "xxx",
```

Comment:
I don't like this, as it introduces an unsafe default just  for developer convenience. 

This should be placed in a dev-only config.